superbuild/CMakeLists.txt

```diff
@@ -37,6 +37,13 @@ set(CCACHE_DIR "default" CACHE STRING "ccache cache directory")
 set(PREBUILT_STAGE2 "NO" CACHE STRING "Provide pre-built stage2 archive to skip pre ispc-stage2 jobs")
 set(PREBUILT_STAGE2_PATH "NO" CACHE STRING "Provide pre-built stage2 toolchain path")
 set(STAGE2_TOOLCHAIN_INSTALL_PREFIX "NO" CACHE STRING "Path to install stage2 Clang/LLVM to")
+set(JOBS "" CACHE STRING "Set the number of jobs for internal build commands (ninja)")
```

Comment:
Is it duplication of `CMAKE_BUILD_PARALLEL_LEVEL` env variable?

Reply:
Possibly, though using this variable does not specifically address the current issue where Ninja will spawn as many threads as there are logical cores available on the system.

Reply:
It contradicts the description of this var: https://cmake.org/cmake/help/latest/envvar/CMAKE_BUILD_PARALLEL_LEVEL.html

Are you defining it as environment variable for the experiment?

Reply:
> Are you defining it as environment variable for the experiment?

Which experiment?

Reply:
> It contradicts the description of this var: https://cmake.org/cmake/help/latest/envvar/CMAKE_BUILD_PARALLEL_LEVEL.html

We can use this value as long as its passed along to ninja for the sub-builds. The sub-builds are not triggered by a call to `cmake --build`, so I'll have to test to see if setting this value obviates the need for `NINJA_JOBS`.

Reply:
```
cmake .. -G Ninja
CMAKE_BUILD_PARALLEL_LEVEL=1 cmake --build .
```
works for me.

Reply:
> ```
> cmake .. -G Ninja
> CMAKE_BUILD_PARALLEL_LEVEL=1 cmake --build .
> ```
> 
> works for me.

This is not working for me, including exporting the value.

Reply:
Try printing `CMAKE_BUILD_PARALLEL_LEVEL` inside CMake to see if it's appears set from CMake stand point.

If we can make it work with standard means, we should do that instead of adding duplicating functionality.

Reply:
The description for the variable is misleading, as setting it from the environment show it to be empty when printing. The value can be passed through the CLI though, which will set ninja's jobs [here](https://github.com/ispc/ispc/pull/2568#discussion_r1266021774).

Reply:
Thats weird, it works for me out of the box when building ISPC without superbuild as I noted above. My CMake version is 3.21. Check if you have relatively new version.